roles/base_patch/tasks/main.yml

```diff
@@ -6,13 +6,6 @@
 - name: "fix sudoers file"
   import_tasks: ./fix_sudoers.yml
 
-- name: "clean up /root for dataservers"
```

Comment:
What is this for?

Reply:
I think this was meant to clean out log files on long-lived "data" instances. We don't have long-lived or data-specific ec2 instances anymore, so this can go.

Reply:
A bit more context: this lived near our invocation of `yum update-minimal ...` and I just happened to be working in the file to notice "clean up /root for dataservers" so I removed it. 😎